pages/toolkits/customer-support/zendesk.mdx

```diff
@@ -0,0 +1,194 @@
+# Zendesk
+
+import ToolInfo from "@/components/ToolInfo";
+import Badges from "@/components/Badges";
+import TabbedCodeBlock from "@/components/TabbedCodeBlock";
+import TableOfContents from "@/components/TableOfContents";
+import ToolFooter from "@/components/ToolFooter";
+
+<ToolInfo
+  description="Enable agents to interact with Zendesk"
+  author="Arcade"
+  codeLink="https://github.com/ArcadeAI/arcade-ai/tree/main/toolkits/zendesk"
+  authType="OAuth2"
+  versions={["0.1.0"]}
+/>
+
+<Badges repo="arcadeai/arcade_zendesk" />
+
+The Zendesk toolkit provides a comprehensive set of tools for managing customer support tickets and knowledge base articles. With this toolkit, users can:
+
+- List and paginate through tickets in their Zendesk account.
+- Retrieve all comments for specific tickets, including the original description and conversation history.
+- Add comments to existing tickets to facilitate communication.
+- Mark tickets as solved, optionally including a final comment.
+- Search for published Help Center articles in the knowledge base, with support for multiple filters in a single request.
+
+This toolkit streamlines the process of handling customer inquiries and accessing support resources.
+
+## Available Tools
+
+<TableOfContents
+  headers={["Tool Name", "Description"]}
+  data={
+    [
+      ["Zendesk.ListTickets", "List tickets from your Zendesk account with offset-based pagination."],
+      ["Zendesk.GetTicketComments", "Get all comments for a specific Zendesk ticket, including the original description."],
+      ["Zendesk.AddTicketComment", "Add a comment to an existing Zendesk ticket."],
+      ["Zendesk.MarkTicketSolved", "Mark a Zendesk ticket as solved, optionally with a final comment."],
+      ["Zendesk.SearchArticles", "Search for Help Center articles in your Zendesk knowledge base."],
+    ]
+  }
+/>
+
+<Tip>
+  If you need to perform an action that's not listed here, you can [get in touch
+  with us](mailto:contact@arcade.dev) to request a new tool, or [create your
+  own tools](/home/build-tools/create-a-toolkit).
+</Tip>
```

Comment:
nit: This should be directing folks to our Github Discussions now.  We can fix all these links in a later PR